src/scenes/level-one/level-one.ts

```diff
@@ -1,7 +1,62 @@
 import * as ex from 'excalibur';
+import { Paddle } from '../../actors/player/paddle';
+import { Ball } from '../../actors/ball';
+import { Brick } from '../../actors/brick';
+import * as Events from "excalibur/dist/Events";
 
 export class LevelOne extends ex.Scene {
-  public onInitialize(engine: ex.Engine) {}
+
+  public bricks: Array<Brick> = [];
+
+  private  _padding: number = 20;
+  private _xoffset: number = 100;
+  private _yoffset: number = 20;
+  private _columns: number = 5;
+  private _rows: number = 3;
```

Comment:
I think linter will say that there should be spaces between class variables, not 100% sure though